src/sentry/api/urls.py

```diff
@@ -333,1067 +356,1085 @@
+            name='sentry-api-0-user-authenticator-enroll'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/authenticators/(?P<auth_id>[^\/]+)/(?P<interface_device_id>[^\/]+)/$',
+            UserAuthenticatorDetailsEndpoint.as_view(),
+            name='sentry-api-0-user-authenticator-device-details'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/authenticators/(?P<auth_id>[^\/]+)/$',
+            UserAuthenticatorDetailsEndpoint.as_view(),
+            name='sentry-api-0-user-authenticator-details'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/emails/$',
+            UserEmailsEndpoint.as_view(),
+            name='sentry-api-0-user-emails'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/emails/confirm/$',
+            UserEmailsConfirmEndpoint.as_view(),
+            name='sentry-api-0-user-emails-confirm'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/identities/(?P<identity_id>[^\/]+)/$',
+            UserIdentityDetailsEndpoint.as_view(),
+            name='sentry-api-0-user-identity-details'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/ips/$',
+            UserIPsEndpoint.as_view(),
+            name='sentry-api-0-user-ips'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/organizations/$',
+            UserOrganizationsEndpoint.as_view(),
+            name='sentry-api-0-user-organizations'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/notifications/$',
+            UserNotificationDetailsEndpoint.as_view(),
+            name='sentry-api-0-user-notifications'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/password/$',
+            UserPasswordEndpoint.as_view(),
+            name='sentry-api-0-user-password'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/notifications/(?P<notification_type>[^\/]+)/$',
+            UserNotificationFineTuningEndpoint.as_view(),
+            name='sentry-api-0-user-notifications-fine-tuning'
+        ),
+        url(
+            r'^(?P<user_id>[^\/]+)/social-identities/$',
+            UserSocialIdentitiesIndexEndpoint.as_view(),
+            name='sentry-api-0-user-social-identities-index'),
+        url(
+            r'^(?P<user_id>[^\/]+)/social-identities/(?P<identity_id>[^\/]+)/$',
+            UserSocialIdentityDetailsEndpoint.as_view(),
+            name='sentry-api-0-user-social-identity-details'),
+        url(
+            r'^(?P<user_id>[^\/]+)/subscriptions/$',
+            UserSubscriptionsEndpoint.as_view(),
+            name='sentry-api-0-user-subscriptions'
+        ),
+    ])),
+
+    # Organizations
+    url(r'^organizations/', include([
+        url(r'^$', OrganizationIndexEndpoint.as_view(), name='sentry-api-0-organizations'),
+        url(
+            r'^(?P<organization_slug>[^\/]+)/$',
+            OrganizationDetailsEndpoint.as_view(),
+            name='sentry-api-0-organization-details'
+        ),
+
+        # Incidents
+        url(
```

Comment:
Does it make sense to include incidents as a sub pattern? ( I assume you can have `^(?P<organization_slug>[^\/]+)/incidents/` as another level)

Reply:
Yeah we could totally do more levels I think. I just didn't.